### ISSUE TITLE
yolov8 training necklace recognition model, resulting in misrecognition, how can I optimize it?

### ISSUE THREAD

**Issue by chaoweiyi** [ultralytics/ultralytics#17122]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

yolov8 training necklace recognition model, resulting in misrecognition, how can I optimize it?
(1) Modify a certain training parameter, which I am not familiar with
(2) Increase training samples, are there any negative samples that can be added? I did not find any
(3) Other ways to optimize
Thank you for your help!

![sx9kvaKDld](https://github.com/user-attachments/assets/9da0c285-7fa3-4361-a172-3bae44adacda)

**Comment by UltralyticsAssistant** [comment]
👋 Hello @chaoweiyi, thank you for reaching out about optimizing your YOLOv8 model for necklace recognition 🚀! This is an automated response, but don't worry—a dedicated Ultralytics engineer will follow up to assist you further soon.

For common optimization strategies, we suggest visiting our [Model Training Tips](https://docs.ultralytics.com/guides/model-training-tips/) to explore various techniques. Increasing your training dataset with diverse examples, including negative samples, can often lead to better model performance. Additionally, experimenting with training parameters like learning rate, batch size, and epochs can also help refine your model. If you're unsure about which parameters to adjust, the documentation might offer helpful guidance.

As you've submitted a question related to custom training, please consider sharing more details, such as examples of your dataset and training logs, to enhance the support you'll receive.

If this is more related to a bug 🐛, providing a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) would significantly aid in diagnosing the issue.

### Community & Resources
Feel free to engage with the Ultralytics community for more personalized advice:
- Join our [Discord](https://ultralytics.com/discord) chat for real-time support 🎧.
- Participate in discussions on [Discourse](https://community.ultralytics.com).
- Check out our [Subreddit](https://reddit.com/r/ultralytics) for sharing and learning from others' experiences.

### Upgrade 
Ensure that your environment is up-to-date! Upgrade to the latest `ultralytics` package and all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) using:

```bash
pip install -U ultralytics
```

### Verified Environments
Run YOLOv8 in any of these verified environments with dependencies preinstalled:
- **Notebooks**: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud**: Use our [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon AWS**: Follow the [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker**: Check the [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

### Status
Our CI tests verify correct operation of YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/). Check the latest build status here: <a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

We're here to help and excited to see your project's progress!

**Comment by Skillnoob** [comment]
Can you share your training code and the `labels.jpg` that gets generated in your training run folder?

**Comment by chaoweiyi** [comment]
![labels](https://github.com/user-attachments/assets/85fac171-6030-4941-875b-3cf9a1e893d3)

```python
if __name__ == "__main__":
    from pathlib import Path
    from ultralytics import YOLO

    model = YOLO("yolov8n.pt")
    model.info()

    data_info_path = Path(r"E:\datasets\necklace.yolov8\dataset.yaml")

    results = model.train(data=data_info_path, epochs=300, device="cuda:0", batch=4)
```

@Skillnoob
I tried to change the pre-trained model yolov8m.pt, and the problem no longer occurred, but I want to try other methods.

**Comment by Skillnoob** [comment]
Your dataset is tiny, ~100 instances is pretty much nothing. You should increase your dataset in size a lot.
Here's a simple guide for minimum requirements of a dataset to achieve decent results.
https://docs.ultralytics.com/modes/export/

**Comment by chaoweiyi** [comment]
ok, that helps me a lot, thank you! @Skillnoob
